### PR TITLE
DOC: Redirect to current location in docs transferred wiki links

### PR DESCRIPTION
**Pull request by jhlegarreta**
Redirect the transferred wiki links to their corresponding current locations in the `docs` folder.

## PR Checklist
- [X] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [X] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

### CODE REVIEW

File: Documentation/docs/learn/faq.md [hunk 1]
```diff
@@ -346,7 +346,7 @@ be summarized as follows:
  guidelines for the development of software-related medical devices."
 ```
 
-For mode details see the page [FDA Guidelines for Software Developement](https://itk.org/Wiki/FDA_Guidelines_for_Software_Development).
+For mode details see the page [FDA Guidelines for Software Developement](fda_off_the_shelf_sw_guidelines.md).
```
Comment (N-Dekker) [inline Documentation/docs/learn/faq.md:349]: Thanks Jon. While the intention is fine to me, I still find this link a bit confusing. It seems to me that the .md file should have been named something like `fda_guidelines_for_software_development.md` instead, without the term "off the shelf". Looking at the original, https://itk.org/Wiki/FDA_Guidelines_for_Software_Development, the title is "FDA Guidelines for Software Development". (It has an "off the shelf" section, but it has other sections as well.)

Comment (jhlegarreta) [reply Documentation/docs/learn/faq.md:349]: OK, but this patch set removes redirections to the wiki for the integrated documents; I will rename the file and change the hyperlink in a separate PR.

**Review by thewtex — APPROVED** [review]
:100:

File: Documentation/docs/learn/faq.md (hunk 1 above)
Comment (jhlegarreta) [reply Documentation/docs/learn/faq.md:349]: @N-Dekker PR #4374.

**Pull request merged**